src/Tools/MicroBuild/Build.proj

```diff
@@ -25,6 +25,8 @@
 
     <MSBuild Projects="$(ProjectDir)Roslyn.sln" BuildInParallel="true" Properties="DeployExtension=false" />
 
+    <Exec Command="$(NuGetPackageRoot)\GitLink\$(GitLinkVersion)\lib\net45\GitLink.exe $(ProjectDir) -u https://github.com/dotnet/roslyn -f Roslyn.sln -c $(Configuration)" />
```

Comment:
Let's chat about this in person.  The perf issue is no longer relevant with GitLink.  But I'm not sure it's going to have the effect we need.